.github/workflows/deploy_to_production.yml

```diff
@@ -1,12 +1,15 @@
-name: Promote to production
+name: Deploy to Sandbox
```

Comment:
Shouldn't this be `Production` instead of `Sandbox`?

Reply:
Yes it should, thank you.